docs/docs/get_started.md

```diff
@@ -2,39 +2,34 @@
 
 ## Clone the project
 
-The [arlas_playground](https://github.com/gisaia/arlas_playground) project contains scripts and resources you need to run the tutorials.
+The **arlas_playground** ([repo](https://github.com/gisaia/arlas_playground){:target="_blank"}) project contains scripts and resources you need to run the tutorials.
 
-Clone it in your working directory
+Clone it in your working directory:
 
 ```shell
 git clone git@github.com:gisaia/arlas_playground.git
 cd arlas_playground
 ```
 
+!!! tip
```

Comment:
this could be add in a section below instead of here. ("Python virtual environment")